website/static/js/pages/support-page.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Code for interaction in the support page
+ */
+
+'use strict';
+
+var Raven = require('raven-js');
+var $ = require('jquery');
+var $osf = require('js/osfHelpers');
+
+$(document).ready(function(){
+    /**
+     * Toggle individual item with jQuery
+     * @param item {Object} jQuery object for the .support-item element
+     * @param turnOff {Boolean} true if we are closing and open item
+     * @private
+     */
+    function _toggleItem (item, turnOff) {
+        var body = item.children('.support-body');
+        var head = item.children('.support-head');
+        var icon = head.children('.fa');
+        if (turnOff){
+            body.slideUp();
+            icon.removeClass('fa-angle-down').addClass('fa-angle-right');
+            item.removeClass('open').addClass('collapsed');
+        } else {
+            body.slideDown();
+            icon.removeClass('fa-angle-right').addClass('fa-angle-down');
+            item.removeClass('collapsed').addClass('open');
+        }
+    }
+
+    function resetFilter () {
+        $('.support-item').each(function() {
+            var el = $(this);
+            _toggleItem(el, true);
```

Comment:
To allow this behavior, the ` turnOff ` flag can remain there, and have it turnOff if flag is true, then do toggling behavior as suggested above otherwise. Does that seem like an improvement?